Bottom: 7b690d3ebb08061bca88c0e278c926f0bf9b72e9
Top:    1accd04bc70c13bd47735c242c82a9cc08d112d1
Author: Ian Kent <raven@themaw.net>
Date:   2013-06-07 14:42:37 +0800

autofs-5.0.7 - add debug alert for waitpid in check_nfs_mount_version()

We don't really case if there's no process to wait for but add a debug
log alert for information.


---

diff --git a/lib/mounts.c b/lib/mounts.c
index 7b959b8..a6f560e 100644
--- a/lib/mounts.c
+++ b/lib/mounts.c
@@ -287,7 +287,8 @@ int check_nfs_mount_version(struct nfs_mount_vers *vers,
 			ret = 0;
 	}
 
-	if (waitpid(f, &status, 0) != f) ;
+	if (waitpid(f, &status, 0) != f)
+		debug(LOGOPT_NONE, "no process found to wait for");
 
 	pthread_sigmask(SIG_SETMASK, &oldsig, NULL);
 	pthread_setcancelstate(cancel_state, NULL);
